Okta/OktaAppAuth.swift

```diff
@@ -38,15 +46,44 @@ public func login() -> Login {
     return Login()
 }
 
+public func signOutFromOkta() -> Logout {
+    // Logout for authorization code flow
+    return Logout(idToken: tokens?.idToken)
+}
+
+public func signOutLocally() -> Promise<Void> {
```

Comment:
After some discussion, let's go back to naming this `signOut`. Also, make sure that the developer can pass a bool parameter to disable revocation.

Reply:
Done.